src/app/clients/form/submitFormHelpers.ts

```diff
@@ -16,14 +16,25 @@ const personToFamilyRecordWithoutFamilyId = (person: Person): FamilyDatabaseInse
         gender: person.gender,
         birth_year: person.birthYear,
         birth_month: person.birthMonth ?? null,
+        recorded_as_child: person.recordedAsChild ?? null,
     };
 };
 
-export const getFamilyMembers = (
+export const getFamilyMembersForDatabase = (
     adults: Person[],
     children: Person[]
 ): FamilyDatabaseInsertRecord[] => {
-    const peopleToInsert = children.concat(adults);
+    const peopleToInsert = children
```

Comment:
This isn't necessary for new family members, because they're created with the new field in ClientForm.tsx.
We're not worried about updating pre-existing adults & children to have the flag, so let's remove this change to simplify.